app/Models/RoleNode.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class RoleNode extends Model
+{
+    protected $table = 'role_node';
```

Comment:
Should be alphabetical

```suggestion
    protected $table = 'node_role';
```